src/main/java/Duke.java

```diff
@@ -1,10 +1,51 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private TaskManager manager = new TaskManager();
+
+    public void exit() {
+        System.out.println("------------------------------------");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("------------------------------------");
+        System.out.println();
+    }
+
+    public void run() {
```

Comment:
Similarly, you could consider renaming this method to be more descriptive, as it could be confused for the Java Thread run() method